aries-site/src/examples/templates/FilterControls/FilterCheckBoxGroup.js

```diff
@@ -0,0 +1,54 @@
+import { useContext, useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { Box, CheckBoxGroup, FormField } from 'grommet';
+
+import { FilterContext } from '.';
+
+export const FilterCheckBoxGroup = ({ attr }) => {
+  const { data, filters, setFilters, getFilterOptions } = useContext(
+    FilterContext,
+  );
+  const { property, header, sort = true, convertToString } = attr;
+  const [value, setValue] = useState(filters[property]);
+  let options = getFilterOptions(data, property);
+
+  if (sort)
+    options.sort((a, b) => {
+      if (a < b) return -1;
+      if (a > b) return 1;
+      return 0;
+    });
+  if (convertToString) options = options.map(option => option.toString());
+
+  useEffect(() => {
+    setValue(filters[property] || []);
+  }, [filters, property, setValue]);
+
+  return (
+    <Box flex={false}>
+      <FormField htmlFor={property} name={property} label={header}>
```

Comment:
Any opportunity to change `header` --> `label` ? Seems a little misleading since header is typically a layout element

Reply:
yes, good call. This naming carried through from a table column definition use case, but doesn't make sense here. Updating.